### PR TITLE
add support for passing tolerations to log aggregators

### PR DESCRIPTION
**Pull request by paynejacob**
for: https://github.com/rancher/rancher/issues/20565

Added support for passing tolerations to logging aggregators.

### CODE REVIEW

File: charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml [hunk 1]
```diff
@@ -28,6 +28,8 @@ annotations: {}
 tolerations:
   - effect: NoSchedule
     operator: Exists
+  - effect: NoExecute
+    operator: Exists
```
Comment (cbron) [inline charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml]: Whats purpose of adding this ?

File: charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml [hunk 2]
```diff
@@ -41,12 +43,6 @@ service:
 cluster:
   dockerRoot: /var/lib/docker
 
-service:
-  ports:
-    - name: metric
-      port: 24231
-      targetPort: metric
-
```
Comment (cbron) [inline charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml]: Why removing metric service ?

**Review by cbron — CHANGES_REQUESTED** [review]
* Does this need to happen in windows too ? 
* We never change charts in-place, we always do a copy of last chart, then commit, then any changes in 2nd commit.

File: charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml (hunk 1 above)
Comment (paynejacob) [reply charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml]: This was from when I was testing I will remove it

File: charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml (hunk 2 above)
Comment (paynejacob) [reply charts/rancher-logging/0.2.0/charts/fluentd/charts/fluentd-linux/values.yaml]: again testing that should be removed

**Comment by paynejacob** [comment]
This functionality exists in logging v2 on rancher 2.5.0. This version of logging is deprecated.

**Pull request closed without merging**